server/src/main/java/com/hedera/block/server/BlockStreamService.java

```diff
@@ -87,6 +95,7 @@ public String serviceName() {
     public void update(final Routing routing) {
         routing.bidi(CLIENT_STREAMING_METHOD_NAME, this::streamSink);
         routing.bidi(SERVER_STREAMING_METHOD_NAME, this::streamSource);
+        routing.unary("GetBlock", this::getBlock);
```

Comment:
We should use a Constant as the ones above to avoid having hardcoded values for the method names